elements/pl-file-editor/pl-file-editor.mustache

```diff
@@ -11,7 +11,8 @@ $(function() {
         minLines: {{#min_lines}}{{min_lines}}{{/min_lines}}{{^min_lines}}undefined{{/min_lines}},
         maxLines: {{#max_lines}}{{max_lines}}{{/max_lines}}{{^max_lines}}undefined{{/max_lines}},
         autoResize: {{#auto_resize}}{{auto_resize}}{{/auto_resize}}{{^auto_resize}}undefined{{/auto_resize}},
-        preview: {{#preview}}"{{preview}}"{{/preview}}{{^preview}}undefined{{/preview}}
+        preview: {{#preview}}"{{preview}}"{{/preview}}{{^preview}}undefined{{/preview}},
+        plOptionFocus: {{#focus}}{{focus}}{{/focus}}{{^focus}}undefined{{/focus}}
```

Comment:
```suggestion
        plOptionFocus: {{focus}}
```

From the code it looks like `focus` must always be the string `"true"` or `"false"`, so we don't need to test here. The same comment actually applies above to `autoResize`.

Reply:
Would it be preferable to also simplify the `autoResize` case then? I was just trying to follow the existing style.

Reply:
I'm all for cleaning up the existing code, I'm guessing that many of the attributes above can be simplified as well.

Reply:
We can remove the extra cases from the mustache but this will leave the burden on the Python side to sanitize all the variables adequately. Is this how it is for most of the other elements or would someone in the future get in trouble without these extra checks?

Reply:
I would think anything coming from the Python side should be sanitized anyway, since we check attributes and assign a default if they aren't defined.

Reply:
I simplified `plOptionFocus` and `autoResize` in this way but I was hesitant for the others without having the time to test them thoroughly. Some of them have `Νone` as the default on the Python side, which would cause the quoted values to be set as `"None"` (e.g. `aceMode: "None"`) and I'm not sure how well Ace Editor even tolerates that now:

```javascript
    if (options.aceMode) {
        this.editor.getSession().setMode(options.aceMode);
    }
```

This is doing `setMode("None")` in that case. Apparently it has been working. Should this be something like:

```javascript
    if (options.aceMode && options.aceMode != "None") {
        this.editor.getSession().setMode(options.aceMode);
    }
```

Reply:
It looks like Ace Editor ends up expecting the default to be `undefined` or `false` and consequently sets the mode to `"ace/mode/text"`:

https://github.com/ajaxorg/ace/blob/5524d248e3f82a70539bc0fbcded626b1c48b7f7/lib/ace/edit_session.js#L861

Since we set it to `"None"` when not specified, it will end up trying to load the module from Ace anyway and have an error that gets discarded and the mode is set to plain text anyway. It's good that Ace handles this gracefully but I think we could just avoid passing it options that it will fail on in that case. Maybe a separate issue to consider though?

Reply:
No worries, I think that this already looks nicer.

Reply:
OK, I'll let it be for now then.